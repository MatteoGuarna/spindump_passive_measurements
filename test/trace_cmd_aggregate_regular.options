--format json --textual --aggregate tag=google 10.30.0.0/24 172.0.0.0/8 --aggregate tag=all 10.30.0.0/24 0.0.0.0/0 --aggregate-mode
