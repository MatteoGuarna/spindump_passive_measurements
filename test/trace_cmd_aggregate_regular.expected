[
{ "Event": "new", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","172.0.0.0/8"], "Session": "0 sessions", "State": "Static", "Tags": "google", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "new", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "0 sessions", "State": "Static", "Tags": "all", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","172.0.0.0/8"], "Session": "1 sessions", "Ts": 1585763140692970, "State": "Static", "Tags": "google", "Right_rtt": 9699, "Packets1": 1, "Packets2": 0, "Bytes1": 84, "Bytes2": 0, "Bandwidth1": 84, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "1 sessions", "Ts": 1585763140692970, "State": "Static", "Tags": "all", "Right_rtt": 9699, "Packets1": 1, "Packets2": 0, "Bytes1": 84, "Bytes2": 0, "Bandwidth1": 84, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","172.0.0.0/8"], "Session": "1 sessions", "Ts": 1585763141722317, "State": "Static", "Tags": "google", "Right_rtt": 38763, "Packets1": 2, "Packets2": 1, "Bytes1": 168, "Bytes2": 84, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "1 sessions", "Ts": 1585763141722317, "State": "Static", "Tags": "all", "Right_rtt": 38763, "Packets1": 2, "Packets2": 1, "Bytes1": 168, "Bytes2": 84, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","172.0.0.0/8"], "Session": "1 sessions", "Ts": 1585763142693933, "State": "Static", "Tags": "google", "Right_rtt": 10244, "Packets1": 3, "Packets2": 2, "Bytes1": 252, "Bytes2": 168, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "1 sessions", "Ts": 1585763142693933, "State": "Static", "Tags": "all", "Right_rtt": 10244, "Packets1": 3, "Packets2": 2, "Bytes1": 252, "Bytes2": 168, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","172.0.0.0/8"], "Session": "1 sessions", "Ts": 1585763143695847, "State": "Static", "Tags": "google", "Right_rtt": 11121, "Packets1": 4, "Packets2": 3, "Bytes1": 336, "Bytes2": 252, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "1 sessions", "Ts": 1585763143695847, "State": "Static", "Tags": "all", "Right_rtt": 11121, "Packets1": 4, "Packets2": 3, "Bytes1": 336, "Bytes2": 252, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","172.0.0.0/8"], "Session": "1 sessions", "Ts": 1585763144718857, "State": "Static", "Tags": "google", "Right_rtt": 33140, "Packets1": 5, "Packets2": 4, "Bytes1": 420, "Bytes2": 336, "Bandwidth1": 84, "Bandwidth2": 168 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "1 sessions", "Ts": 1585763144718857, "State": "Static", "Tags": "all", "Right_rtt": 33140, "Packets1": 5, "Packets2": 4, "Bytes1": 420, "Bytes2": 336, "Bandwidth1": 84, "Bandwidth2": 168 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763147472777, "State": "Static", "Tags": "all", "Right_rtt": 66084, "Packets1": 6, "Packets2": 5, "Bytes1": 504, "Bytes2": 420, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763148468786, "State": "Static", "Tags": "all", "Right_rtt": 61863, "Packets1": 7, "Packets2": 6, "Bytes1": 588, "Bytes2": 504, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763149468909, "State": "Static", "Tags": "all", "Right_rtt": 61481, "Packets1": 8, "Packets2": 7, "Bytes1": 672, "Bytes2": 588, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763150472783, "State": "Static", "Tags": "all", "Right_rtt": 61091, "Packets1": 9, "Packets2": 8, "Bytes1": 756, "Bytes2": 672, "Bandwidth1": 84, "Bandwidth2": 168 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763151474775, "State": "Static", "Tags": "all", "Right_rtt": 60254, "Packets1": 10, "Packets2": 9, "Bytes1": 840, "Bytes2": 756, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763153483463, "State": "Static", "Tags": "all", "Right_rtt": 61225, "Packets1": 12, "Packets2": 10, "Bytes1": 1008, "Bytes2": 840, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763154488043, "State": "Static", "Tags": "all", "Right_rtt": 60784, "Packets1": 13, "Packets2": 11, "Bytes1": 1092, "Bytes2": 924, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763155488301, "State": "Static", "Tags": "all", "Right_rtt": 59789, "Packets1": 14, "Packets2": 12, "Bytes1": 1176, "Bytes2": 1008, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763156494219, "State": "Static", "Tags": "all", "Right_rtt": 61165, "Packets1": 15, "Packets2": 13, "Bytes1": 1260, "Bytes2": 1092, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763157498232, "State": "Static", "Tags": "all", "Right_rtt": 61502, "Packets1": 16, "Packets2": 14, "Bytes1": 1344, "Bytes2": 1176, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763158498553, "State": "Static", "Tags": "all", "Right_rtt": 60861, "Packets1": 17, "Packets2": 15, "Bytes1": 1428, "Bytes2": 1260, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763159502658, "State": "Static", "Tags": "all", "Right_rtt": 60751, "Packets1": 18, "Packets2": 16, "Bytes1": 1512, "Bytes2": 1344, "Bandwidth1": 84, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "NET2NET", "Addrs": ["10.30.0.0/24","0.0.0.0/0"], "Session": "2 sessions", "Ts": 1585763160506022, "State": "Static", "Tags": "all", "Right_rtt": 61063, "Packets1": 19, "Packets2": 17, "Bytes1": 1596, "Bytes2": 1428, "Bandwidth1": 84, "Bandwidth2": 84 }
]
